.drone.yml

```
@@ -7,6 +7,11 @@ platform:
   arch: amd64
 
 steps:
+- name: build-go-apps
```

We only have Go applications at the moment but even if we have other applications that would need compilation, we should have this generic since the `make all` will account for the underlying non-Go application.

```suggestion
- name: build
```